src/lib/data/buyables/frem.ts

```diff
@@ -0,0 +1,82 @@
+import { resolveNameBank } from '../../util';
+import { Buyable } from './buyables';
+
+export const fremennikClothes: Buyable[] = [
```

Comment:
these should have a qp requirement since you have to complete the fremennik trials before she will speak to you, Its a med level quest so something like 75 qp sounds appropriate to me.